modules/monitoring-configuring-notifications-for-default-platform-alerts.adoc

```diff
@@ -0,0 +1,134 @@
+// Module included in the following assemblies:
+//
+// * observability/monitoring/managing-alerts.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-notifications-for-default-platform-alerts_{context}"]
+= Configuring notifications for default platform alerts
+
+You can customize where and how Alertmanager sends notifications about default platform alerts.
+You can achieve this by editing the default configuration in the `alertmanager-main` secret in the `openshift-monitoring` namespace.
+
+[IMPORTANT]
+====
+The default configuration of the `alertmanager-main` secret does not generate notifications. Therefore, it is recommended to configure Alertmanager to receive notifications, as outlined in this section.
+====
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` cluster role.
+
```

Comment:
for consistency as https://79966--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/monitoring/managing-alerts.html#configuring-notifications-for-user-defined-alerts_managing-alerts, it's better to add below prerequisite
You have installed the OpenShift CLI (oc).